nautobot/core/constants.py

```diff
@@ -32,6 +32,24 @@
 
 FILTER_NEGATION_LOOKUP_MAP = {"n": "exact"}
 
+FILTER_LOCAL_FIELDS = {
+    "BigIntegerField",
+    "CharField",
+    "DateField",
+    "DateTimeField",
+    "DecimalField",
+    "FloatField",
+    "IntegerField",
+    "PositiveIntegerField",
+    "PositiveSmallIntegerField",
+    "SlugField",
+    "SmallIntegerField",
+    "TextField",
+    "UUIDField",
+}
+
+FILTER_LOOKUP_EXPR = {"isnull"}
```

Comment:
These constants appear to be ambiguously named to me when they are currently only used to determine if a field should support `isnull` filtering. Were you thinking there would be future extensions to this?